include/crm/common/output_internal.h

```diff
@@ -28,7 +28,7 @@ extern "C" {
  */
 
 
-#  define PCMK__API_VERSION "2.24"
+#  define PCMK__API_VERSION "2.25"
```

Comment:
There have been a few updates since this was started, so you'll need to rebase on main and bump this to 2.29